vendor/github.com/containers/common/pkg/auth/cli.go

```diff
@@ -0,0 +1,41 @@
+package auth
+
+import "github.com/spf13/pflag"
+
+// LoginOptions represents the results for flags in login
+type LoginOptions struct {
+	Authfile      string
+	CertDir       string
+	Password      string
+	Username      string
+	StdinPassword bool
+	TlsVerify     bool
+	GetLogin      bool
```

Comment:
If the distinction needs to exist at all, this should at the very least clearly document which options are handled by this subpackage and which need to be handled by the caller.